Makefile

```diff
@@ -124,6 +124,9 @@ clean:
 distclean: clean
 	rm -rf vendor/
 
+cl:
+	@go run cmd/sdkch/main.go add -i
```

Comment:
Didn't we say we'd remove this in favor of interactive mode?

Reply:
lol now I'm confused... this _is_ the interactive mode. If you're referring to the ability to have `make cl` I think we ought to still keep it for ease of use in the PR template - however this is totally in complement to using `sdkch` as is (we still build it with `make tools` right)

Reply:
OK got it now.

Let's set our expectations adequately then: what do we expect from our contributors? What should they {be able to,} do?
Do we expect them at least to run `make tools` first? I always believed that that was a sane expectation: if you want to contribute to X's project, you will be kindly asked to comply w/ their coding style guidelines and learn about their tooling. It's easy to comply with our coding style (it's just standard Go) as well as build our tools (`make tools`) and learn about them (`sdkch --help`).

After you run `make tools`, you have `sdkch` in your `$PATH`, thus the `cl` target is redundant.

Reply:
Okay my compromise on this issue would be both delete this make entry but also rename `sdkch` to something simple like `cl` (ultimately we would need to change the name to not include `sdk` in it if we hope other repos such as tendermint to be using it!)

Reply:
Yeah, the gaia repo will certainly use it too. But let's pick a good name, shall we? In Debian we have a similar (yet different) tool called `dch`. It stands for _Debian Changelog_. What about `cch` as in _Cosmos Changelog_?